CODEOWNERS

```
@@ -0,0 +1 @@
+* @github/web-systems
```

We probably want the reviewers team, right? Looks like that's what we use in other OSS like the [eslint plugin](https://github.com/github/eslint-plugin-github/blob/main/CODEOWNERS)

```suggestion
* @github/web-systems-reviewers
```